### PR TITLE
add possibility to have a prefixMessage in discord notification before the embed

### PR DESCRIPTION
**Pull request by Revyn112**
This pull request add a new form element to the notification setup for discord.
You can enter a prefixMessage and it will be added as content message before the embed.

This is useful when you want to notify certain usergroups about the event.

### CODE REVIEW

**Comment by Revyn112** [comment]
Setup: https://img.revyn112.dev/data/firefox_rixwt1U90N.png
Output: https://img.revyn112.dev/data/Discord_eI5vgSfCzG.png

**Comment by Saibamen** [comment]
### Setup
image

### Output
image

**Review by Saibamen — APPROVED** [review]

File: src/components/NotificationDialog.vue [hunk 1]
```diff
@@ -80,6 +80,11 @@
                                 <label for="discord-username" class="form-label">Bot Display Name</label>
                                 <input id="discord-username" v-model="notification.discordUsername" type="text" class="form-control" autocomplete="false" :placeholder="$root.appName">
                             </div>
+
+                            <div class="mb-3">
+                                <label for="discord-prefix-message" class="form-label">Prefix</label>
```
Comment (louislam) [inline src/components/NotificationDialog.vue:85]: Prefix Custom Message would be better?

Comment (Revyn112) [reply src/components/NotificationDialog.vue:85]: yeah you're right

**Pull request merged**